AbbyM/Lab1/include/hello_world.h

```diff
@@ -0,0 +1,11 @@
+#ifndef hello_world_h
+#define hello_world_h
+
+#include <string>
+
```

Comment:
where is your iostream include? also why include string when you dont use it?

please include what you use: https://www.fluentcpp.com/2021/01/01/include-what-you-use/